src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -353,21 +404,22 @@ private boolean isSearchCaseReferenceTypePresent(CaseDetails exceptionRecord) {
      * Information received in the callback call for the attach event.
      */
     private static class AttachToCaseEventData {
-        public final String exceptionRecordJurisdiction;
-        public final String service;
-        public final String targetCaseRef;
-        public final String targetCaseRefType;
-        public final Long exceptionRecordId;
-        public final List<Map<String, Object>> exceptionRecordDocuments;
-        public final String idamToken;
-        public final String userId;
-
-        public AttachToCaseEventData(
+        final String exceptionRecordJurisdiction;
+        final String service;
+        final String targetCaseRef;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4558409)